static/js/settings.js

```diff
@@ -129,8 +129,12 @@ exports.launch = function (section) {
     settings_sections.reset_sections();
 
     overlays.open_settings();
-    settings_panel_menu.normal_settings.activate_section(section);
-    settings_toggle.highlight_toggle('settings');
+    const isMobile = /iPhone|iPad|iPod|Android/i.test(navigator.userAgent);
```

Comment:
We should use `util.isMobile` for this check.  We can probably simplify the regex in `util.isMobile` to exclude things like "Blackberry", although they are probably harmless.  But regardless, we want to do the details of that check in only one place.

Reply:
Sure @showell thanks for the suggestion :)